### PR TITLE
Reword assert message

### PR DESCRIPTION
**Pull request by atbrakhi**
Ms2ger is not actively involved in the project. This PR rewords the assert message and removes names to ensure the responsibility is not placed on a single person

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors

### CODE REVIEW

File: tests/wpt/mozilla/tests/mozilla/interfaces.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ function test_interfaces(interfaceNamesInGlobalScope) {
       assert_true(name in interfaceMap,
                   "If this is failing: DANGER, are you sure you want to expose the new " +
                   "interface " + name + " to all webpages as a property on the global? " +
-                  "Do not make a change to this file without review from jdm or Ms2ger " +
+                  "Do not make a change to this file without review" +
```
Comment (mrobinson) [inline tests/wpt/mozilla/tests/mozilla/interfaces.js:83]: All changes need a review, so this makes the statement a bit redundant. Maybe we should agree on a policy for when to expose new interfaces and write that here?

**Review by mrobinson — CHANGES_REQUESTED** [review]

File: tests/wpt/mozilla/tests/mozilla/interfaces.js (hunk 1 above)
Comment (atbrakhi) [reply tests/wpt/mozilla/tests/mozilla/interfaces.js:83]: I think we can do that separately and mention it in the Servo book or somewhere else.

For the assert message, maybe I can change it to just:

```
"If this is failing: DANGER, are you sure you want to expose the new " +
interface " + name + " to all webpages as a property on the global?"
```

What do you think? I'm not sure if we want to provide a policy in the assert message anyway.

Comment (sagudev) [reply tests/wpt/mozilla/tests/mozilla/interfaces.js:83]: We should use CODEOWNERS to ensure knowledgeable people review such PR.

Comment (jdm) [reply tests/wpt/mozilla/tests/mozilla/interfaces.js:83]: > I think we can do that separately and mention it in the Servo book or somewhere.
> 
> For the assert message, maybe I can change it to just:
> 
> ```
> "If this is failing: DANGER, are you sure you want to expose the new " +
> interface " + name + " to all webpages as a property on the global?"
> ```
> 
> What do you think? I'm not sure if we want to provide a policy in the assert message anyway.

I like this change.

Comment (mrobinson) [reply tests/wpt/mozilla/tests/mozilla/interfaces.js:83]: > What do you think? I'm not sure if we want to provide a policy in the assert message anyway.

Seems fine to me if @jdm is okay as well.

**Review by mrobinson — APPROVED** [review]

**Pull request merged**